examples/browser/package.json

```diff
@@ -39,6 +40,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
+    "bunyan": "^1.8.10",
```

Comment:
@hexa00 
- if something is required at runtime it should go to `dependencies`
- if something is required by npm scripts then it should go to `devDependencies`
- if something is required at runtime and by npm scripts then go to `dependencies` as well